.github/workflows/ci.yaml

```
@@ -28,30 +31,30 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        arch:
-          - amd64
-          - arm
-        os:
-          - ubuntu-24.04
+        platform:
+          - target: ubuntu-24.04
+          - target: ubuntu-24.04-arm
+          - target: windows-latest
+          - target: macos-15-intel
+          - target: macos-15
```

Is this nesting needed? Can't you just do:

```suggestion
        target:
          - ubuntu-24.04
          - ubuntu-24.04-arm
          - windows-latest
          - macos-15-intel
          - macos-15
```

Like with Python?

---

done